src/layers/tile-layer-utils/viewport-utils.js

```diff
@@ -20,12 +20,12 @@ function pixelsToTileIndex(a) {
  */
 export function getTileIndices(viewport, maxZoom, minZoom, tileSize) {
   const z = Math.min(0, Math.ceil(viewport.zoom));
+
   if (z <= minZoom) {
     return [{ x: 0, y: 0, z: minZoom }];
   }
   const bbox = getBoundingBox(viewport);
-  let [minX, minY] = [[bbox[0], z, tileSize], [bbox[1], z, tileSize]].map(pixelsToTileIndex);
-  let [maxX, maxY] = [[bbox[2], z, tileSize], [bbox[3], z, tileSize]].map(pixelsToTileIndex);
+  let [minX, minY, maxX, maxY] = bbox.map(pixels => pixelsToTileIndex(pixels, z, tileSize));
```

Comment:
Still have a preference for using `const`, and using separate names for these, and the variables below.